README.md

```diff
@@ -1,3 +1,96 @@
 <!-- default badges list -->
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
+
+# DevExpress Chat for .NET MAUI
+
+This example demonstrates a Chart control that allows users to send and receive massages.
```

Comment:
I wouldn't say here that the control "allows to send/receive message". The example, as I see, only focuses on building the UI shell. The messages are never sent or received - they are just a collection in the model. No data is transferred between different clients. 

```suggestion
This example uses DevExpress .NET MAUI Components to display a chat view with sender and receiver messages, suggested quick replies, and response input controls.
```